### PR TITLE
fix: add tiled filter dropdown select

### PR DESCRIPTION
**Pull request by laojun**
## What this PR does / why we need it:
add tiled filter dropdown select

## Does this PR introduce a user interface change?

9208BD26-AA12-44FB-B9BC-7515A976EB9C

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |            |
| 🇨🇳 中文    |             |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/common/components/tiled-filter.tsx [hunk 1]
```diff
@@ -187,4 +185,77 @@ const TiledFilter = (props: IProps) => {
...
+
+const SelectFieldItem = (props: IFieldItemProps) => {
+  const { field, labelWidth, value, onChangeItem } = props;
+  const { type, label, options, showSearch, multiple } = field;
+
+  let Comp: React.ReactNode = null;
+  const _value: string[] = multiple ? value : [value];
+  switch (type) {
+    case 'select':
+      Comp = (
```
Comment (McDaddy) [inline shell/app/common/components/tiled-filter.tsx:203]: it seems `Comp` no need Capitalize

Comment (laojun) [reply shell/app/common/components/tiled-filter.tsx:203]: Comp return as a Component or null, Capitalize as a regular Component

**Pull request merged**